### ISSUE TITLE
How to add a TOC in a post explicitely?

### ISSUE THREAD

**Issue by Ciges** [terrylinooo/githuber-md#152]
**Any issue will be ignored if you don't provide the following detailed information.**

**Describe the bug**

I would like to add a TOC only in some posts, not in everyone. Is it possible to tell in a post where to generate a Table of Contents?

Thanks a lot for your great work!

**Comment by noraj** [comment]
This plugin doesn't do that. To do this I disabled teh TOC feature in WP Githuber MD and used another plugin: [easy-table-of-contents](https://wordpress.org/plugins/easy-table-of-contents/)

**Comment by Ciges** [comment]
Hi @noraj !

After looking for the solution I have just made the same. I was looking for a plugin that allows to add a TOC **only if I explicitely want it and where I want**.

And after a quick search I installed and tested with success [Table of Content Plus](https://wordpress.org/plugins/table-of-contents-plus/). 

It's really easy to use it. After installing it just add the word `[toc]` at the place of the post you want it.

Thanks!
